metrics/metrics.go

```diff
@@ -36,14 +36,16 @@ const (
 	MakeAuctionResponse OverheadType = "make-auction-response"
 	// MakeBidderRequests - measures the time needed to fetch a stored request (if needed), parse, unmarshal, and validate the OpenRTB request, interpret its privacy policies, and split it into multiple requests sanitized for each bidder
 	MakeBidderRequests OverheadType = "make-bidder-requests"
+	// MakeHTTPRequest - measures the time needed to receive response back from the bidder server
+	MakeHTTPRequest OverheadType = "make-http-request"
```

Comment:
How about changing `MakeHTTPRequest` to `BidderServerResponse`?

```
// BidderServerResponse - measures the time PBS has to wait to receive a response from the bidder server
BidderServerResponse OverheadType = "bidder-server-response"
```

Reply:
Addressed in [ac955cb](https://github.com/prebid/prebid-server/pull/2722/commits/ac955cb1a9fadfaff53bce36c55fc7d98f7a028b)